server/ServerManager.cpp

```diff
@@ -0,0 +1,255 @@
+#include "ServerManager.hpp"
+
+#include <unistd.h>
+
+#include <exception>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "../core/Kqueue.hpp"
+#include "../core/Socket.hpp"
+
+/**
+ * Constructor & Destructor
+ */
+
+ServerManager::ServerManager(std::string hostIp, int port,
+                             std::vector<Server> configs)
+    : _hostIp(hostIp), _port(port), _configs(configs) {}
```

Comment:
여기서 _serverFd를 -1로 초기화해서 할당되지 않은 상태라고 명시해도 좋을 것 같아요.
```suggestion
ServerManager::ServerManager(std::string hostIp, int port,
                             std::vector<Server> configs)
    : _serverFd(-1), _hostIp(hostIp), _port(port), _configs(configs) {}
```